include/ftof.h

```diff
@@ -0,0 +1,28 @@
+/*! \file ftof.h
+ *  \brief Main header file for Fast/Fenics interface
+ */
+
+
+#ifndef __FTOF_H
+#define __FTOF_H
+
+#include <dolfin.h>
+
+extern "C"
+{
+#include "fasp.h"
+#include "fasp_functs.h"
+    INT fasp_solver_bdcsr_krylov_block_3(block_dCSRmat *A,
+                                       dvector *b,
+                                       dvector *x,
+                                       itsolver_param *itparam,
+                                       AMG_param *amgparam,
+                                       dCSRmat *A_diag);
+#define FASP_BSR     ON  /** use BSR format in fasp */
```

Comment:
Yeah, I didn't check for freed memory since this PR was about converting to FASP.  Also, I think the lowercase convention (using ``_`` to separate words) is good.  The capital letters should only occur when referring to objects such as ``convert_to_dCSRmat`` for example.